fe/fe-core/src/test/java/com/starrocks/sql/plan/InsertPlanTest.java

```diff
@@ -566,4 +566,41 @@ public void testExplainInsert() throws Exception {
         ExecPlan execPlan = new StatementPlanner().plan(statementBase, connectContext);
         Assert.assertTrue(((InsertStmt) statementBase).getQueryStatement().isExplain());
     }
+
+    @Test
+    public void testInsertExchange() throws Exception {
+        {
+            // keysType is DUP_KEYS
+            String sql = "explain insert into t0 select * from t0";
+            String plan = getInsertExecPlan(sql);
+            assertContains(plan, "  OLAP TABLE SINK\n" +
+                    "    TUPLE ID: 1\n" +
+                    "    RANDOM\n" +
+                    "\n" +
+                    "  0:OlapScanNode\n" +
+                    "     TABLE: t0");
+        }
+        {
+            // KesType is PRIMARY_KEYS
+            String sql = "explain insert into tprimary select * from tprimary";
+            String plan = getInsertExecPlan(sql);
+            assertContains(plan, "  STREAM DATA SINK\n" +
+                    "    EXCHANGE ID: 01\n" +
+                    "    HASH_PARTITIONED: 1: pk\n" +
+                    "\n" +
+                    "  0:OlapScanNode\n" +
+                    "     TABLE: tprimary");
+        }
+        {
+            // KesType is AGG_KEYS
+            String sql = "explain insert into test_object select * from test_object";
```

Comment:
Would better add UT to cover the group by sql, to ensure no additional shuffle node.

Reply:
Ok, I'll make up